xbmc/pvr/epg/EpgContainer.h

```diff
@@ -206,6 +199,19 @@ namespace PVR
      */
     int GetFutureDaysToDisplay() const;
 
+    /*!
+     * @brief Inform the epg container that playback of an item just started.
+     * @param item The item that started to play.
+     */
+    void OnPlaybackStarted(const CFileItemPtr &item);
```

Comment:
Do we have an interface for these methods somewhere? Seems like a lot of classes have methods like this.

Reply:
Good point. We do not currently have such an interface.

Reply:
Regarding the naming: how about `IPlaybackStateListener` ? Where to put it file-wise)

Reply:
The name sounds good, but I'm afraid I don't know off the top of my head where to put such an interface.